tests/globalhelper/crd.go

```diff
@@ -0,0 +1,70 @@
+package globalhelper
```

Comment:
Function from this module is used only in observability suite. So better to keep them in observability/helper

Reply:
Done!